contracts/cw-croncat/src/slots.rs

```diff
@@ -214,16 +214,13 @@ impl<'a> CwCroncat<'a> {
         // Need to remove this slot if no hash's left
         if slot_data.is_empty() {
             self.clean_slot(storage, slot, kind);
-        }
-
-        if hash.is_some() {
+        } else {
             store
                 .update(storage, *slot, |_d| -> StdResult<Vec<_>> { Ok(slot_data) })
                 .ok();
```

Comment:
Since we touching this method, I think it's cleaner to rewrite as 
```suggestion
            store.save(storage, *slot, &slot_data).ok()?;
```